### PR TITLE
Create Thilina Gunarathne

### PR DESCRIPTION
**Pull request by biswajithdissanayake**

### CODE REVIEW

**Comment by ishanfdo18098** [comment]
Please remove the space in the file name and add .html at the end

File: pages/staff/academic-staff/Thilina Gunarathne [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+research_interests: [ 'Image Processing', 'Programming Languages', 'Software Engineering' ]
+
+---
+
+<i class="fas fa-user"></i>
+<a><b><u>BIO</u></b></a>
+
+<p>
+    Mr.Thilina Gunarathne  received B.Sc (Eng) in Computer Engineering from University of Vauniya, Sri Lanka in 2020  She joined the University of Peradeniya as a Lecturer (on contract) 2020.
```
Comment (NuwanJ) [inline pages/staff/academic-staff/Thilina Gunarathne:31]: Content-related review is required on the following:
1. `She` joined the 
2. Lecturer (on contract) `2020`.

**Review by biswajithdissanayake — COMMENTED** [review]
Done the changes mentioned by admin

**Comment by biswajithdissanayake** [comment]
Dear Admin,
Could you please make an arrangement to add "postgraduate projects" to the
students project page.
I have attached the screenshot for your reference.
Best Regards
Biswajith Dissanayake

[image: image.png]

**Comment by NuwanJ** [comment]
@biswajithdissanayake can you open an issue on https://github.com/cepdnaclk/projects.ce.pdn.ac.lk/issues with the required details? 

Also refer https://projects.ce.pdn.ac.lk/docs/how-to-add-a-course/ and update the Google Doc with the required information

**Pull request merged**